techniques/failures/F108.html

```diff
@@ -99,21 +99,21 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
 
     
         <section id="description">
-        <h2>Description</h2>
-        <p>The objective of this failure is to avoid situations in which people with motor impairments who cannot operate content because the only way to actuate a function is by dragging a target element from its initial position to some other position. The failure occurs when there is no alternative single pointer input available to actuate the function. The alternative may involve a series of single pointer interactions (for example, activating a target to be moved; opening a dropdown menu; and selecting a drop destination from the list of menu items offered).</p>
+        <h2>解説</h2>
+        <p>この失敗例の目的は、ある機能を実行する唯一の方法が、ターゲット要素を初期位置から別の位置にドラッグすることであるため、運動障害のある人がコンテンツを操作できない状況を回避することである。この失敗例は、機能を実行するための代替となるシングルポインタ入力がない場合に発生する。代替としては、一連のシングルポインタの操作 (例えば、移動対象のアクティブ化、ドロップダウンメニューの表示、表示されたメニュー項目リストからドロップ先の選択など) が挙げられる。</p>
```

Comment:
冒頭文はわかりにくいですね……原文が悪いんですが……。

カッコ内のセミコロンで区切られた3つのものは、例が3つあるものと解釈されているように思いますが、その3つの操作を一連で行うという1つの例のはずです (a series of single pointer interactions の例)。

こんな感じにしてはどうでしょうか。

```suggestion
        <p>この失敗例の目的は、ターゲット要素を初期位置から別の位置にドラッグすることが機能を実行する唯一の方法であることにより、運動障害のある人がコンテンツを操作できない、という状況を回避することである。この失敗例は、機能を実行するための代替となるシングルポインタ入力がない場合に発生する。代替としては、一連のシングルポインタの操作が挙げられる (例えば、移動対象をアクティブ化し、ドロップダウンメニューを表示し、表示されたメニュー項目リストからドロップ先を選択する) 。</p>
```

Reply:
おおむね提案いただいたものでよいと思いますが、becauseの箇所は「ために」としておきたいのですが、どうでしょう

```suggestion
        <p>この失敗例の目的は、ターゲット要素を初期位置から別の位置にドラッグすることが機能を実行する唯一の方法であるために、運動障害のある人がコンテンツを操作できない、という状況を回避することである。この失敗例は、機能を実行するための代替となるシングルポインタ入力がない場合に発生する。代替としては、一連のシングルポインタの操作が挙げられる (例えば、移動対象をアクティブ化し、ドロップダウンメニューを表示し、表示されたメニュー項目リストからドロップ先を選択する)。</p>
```